SPECS/prometheus-adapter/prometheus-adapter.spec

```diff
@@ -28,11 +32,17 @@ make test
 
 %files
 %license LICENSE NOTICE
+%{_bindir}/*
+
+%files docs
```

Comment:
If we don't have this subpackage depend on the main package, then it needs a license file associated with it.